libs/image-editor/ImageEditor/src/main/kotlin/org/wordpress/android/imageeditor/EditImageActivity.kt

```diff
@@ -55,4 +56,11 @@ class EditImageActivity : AppCompatActivity() {
             navController.navigateUp(appBarConfiguration) || super.onSupportNavigateUp()
         }
     }
+
+    override fun onBackPressed() {
+        super.onBackPressed()
+        if (hostFragment.childFragmentManager.backStackEntryCount == 0) {
```

Comment:
I think this is one of the exceptions which won't be easy to move to a VM. I'll keep thinking about other possible solution as this feel a bit like hack.